.github/workflows/test.yml

```diff
@@ -149,6 +175,11 @@ jobs:
         echo "::set-env name=GOPATH::$GOPATH"
         export PATH=$GOPATH/bin:$PATH
         echo "::add-path::$GOPATH/bin"
+  
+    - name: IPv6 Disable firewall
+      if: matrix.ipfamily.ipv6 == true && matrix.ipfamily.ipv4 == false
+      run: |
+        sudo ufw disable
```

Comment:
why do we only want to do this for single-stack IPv6 but not for dual-stack?

Reply:
ok, backing up; why do we need this for single-stack IPv6? Presumably the firewall is blocking something that we need unblocked when using IPv6? Is that something also something we will need when using dual-stack?

Reply:
There are issues with firewalld and IPv6, and in my testing I hit these issues and so I turned off for IPv6. Can't really test dual-stack yet, so it may (probably) have the same issue. But can't really verify until I can test it. I know some people just turn off firewalld all the time, so that was more my question. I think we should leave on for dual-stack until I can verify it needs to be turned off.
 
Some firewalld and IPv6 issues:
https://github.com/firewalld/firewalld/issues/461
https://github.com/kubernetes-sigs/kind/issues/1283
https://github.com/kubernetes-sigs/kind/issues/1257

Reply:
just always disable it, we dont need it for KIND deployments

Reply:
This appears to disable ufw.. not firewalld.